src/Application.CommandServices/Class1.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace PetProjects.MicroTransactionsApi.Application.CommandServices
```

Comment:
I think the same should be ...CommandHandlers instead of CommandServices.

From my point of view the CommandHandler does everything needed for the transaction have success, including doing queries.

With this we will achieve a separation by entity inside the projects. I think iof we add a folder named handlers everything will go there and it will be chaotic. 

Something like this:

PetProjects.MicroTransactionsApi.Application.CommandHandler
                                                      \Transactions
                                                      \Stuff we need

Is cleaner than
         
PetProjects.MicroTransactionsApi.Application.CommandServices
                                                      \Handlers\Transactions
                                                                      \Stuff we need


What do you think?

Reply:
I was thinking of not using the mediator pattern for the execution of queries so in my vision this logic would be inside a Service class which would make this layer more than just Handlers.
Later I decided that it was better to improve the Mediator in framework-cqrs to fulfill my requirements so now I agree with you, I'm going to rename *Services,csproj to *Handlers.csproj removing the Handlers folder and creating the query/command objects in the controller.

Reply:
Done in 6810897